### ISSUE TITLE
Any support to FreeBSD soon?

### ISSUE THREAD

**Issue by mojadita** [go-task/task#1068]
As go is supported in FreeBSD, I was wondering if task can be ported to this operating system.  I just need it to build the arduino-cli platform, and I don't actually want to geet too deep in the subtasks required to build it.  Is there any intention to support it in a near future?

Thanks

**Comment by mojadita** [comment]
I got it! just it took to clone the git repository and build it.  The fault was  mine, that am new to go language, and didn't know the path where it installs the binaries (~/go/bin)  and it was not in the $PATH.  Thanks,  I'll close the issue

**Comment by andreynering** [comment]
Hi @mojadita,

Thanks for commenting and sorry for the wait.

Good that you was able to build from source. I just added FreeBSD to the operating systems to build binaries for: 125f34ef47c5ea138404a7a75c7152df23d564bf. I think its used unoften but the fact is that it's pretty cheap for us to support it (thanks Go for the easy cross-compiling 🙂).

It just missed the v3.24.0 release I just made, though. I'll be available on the next one.
